__tests__/parse.spec.js

```diff
@@ -163,14 +163,14 @@ diff --git a/file1 b/file1
 index 123..456 789
 --- a/file1
 +++ b/file1
-@@ -1,2 +1,2 @@
+@@ -1,1 +1,1 @@
```

Comment:
This and next one tests had invalid values here - diff contained only single lines added/removed.

Reply:
ok